### PR TITLE
Remove screenshare extension support

### PR DESCRIPTION
**Pull request by jallamsetty1**
- Use getDisplayMedia on browsers where it is supported.
- desktopSharingChromeDisabled and desktopSharingFirefoxDisabled will no longer be supported.

### CODE REVIEW

**Comment by jallamsetty1** [comment]
@saghul sorry about all the force pushes, I just keep discovering more and more code that is obsolete. I have tested all clients except for spot which I will do tomorrow.

**Comment by jallamsetty1** [comment]
Jenkins pls test this pls

File: modules/RTC/ScreenObtainer.js [hunk 1]
```diff
@@ -188,8 +117,7 @@ const ScreenObtainer = {
 
             window.JitsiMeetScreenObtainer.openDesktopPicker(
                 {
-                    desktopSharingSources: desktopSharingSources
-                        || this.options.desktopSharingChromeSources
+                    desktopSharingSources
```
Comment (saghul) [inline modules/RTC/ScreenObtainer.js:120]: Since this is now only used in electron, can we hardcode this to `[ 'screen', 'window']` ?

**Comment by jallamsetty1** [comment]
Jenkins pls test this pls

File: modules/RTC/ScreenObtainer.js (hunk 1 above)
Comment (jallamsetty1) [reply modules/RTC/ScreenObtainer.js:120]: in a subsequent PR maybe ? that's what we decided in the standup.

**Pull request merged**